Gems/ROS2/Code/Source/Frame/ROS2FrameComponent.cpp

```diff
@@ -57,6 +59,17 @@ namespace ROS2
             // Found the component!
             return component;
         }
+
+        //! Check whether the entity has a component of the given type
+        //! @param entity pointer to entity
+        //! @param typeId type of the component
+        //! @returns true if entity has component with given type
+        bool CheckIfEntityHasComponentOfType(const AZ::Entity* entity, const AZ::Uuid typeId)
```

Comment:
nit: As this function is local to the .cpp file we can mark it `static` to show it has internal linkage (this should make it possible to inline and will compile slightly faster hopefully! 😅)